### DISCUSSION TITLE
[cds] Full width card

### DISCUSSION THREAD

**Discussion by mathisloge** [vmware/clarity#6155] [Q&A] [answered]
I'm trying the new cds. 
But I'm struggeling to get a simple full width card to work. 
I've expected something like that to work:
```html
<section cds-layout="grid gap:lg">
    <cds-card cds-layout="col:12">
      <div>
        <h2 cds-text="section">Card Title</h2>
        <div cds-text="body light" cds-layout="p-y:lg">
        </div>
        <cds-divider cds-card-remove-margin></cds-divider>
        <div cds-layout="horizontal gap:sm p-y:sm align:vertical-center"></div>
      </div>
    </cds-card>
</section>
```

The red rectangle is the expected size.
![image](https://user-images.githubusercontent.com/13556116/125859803-49dddcf1-434e-4bd9-a379-0dbc0fa8bff6.png)

What am I missing?

**Comment by hippee-lee** [comment, marked as answer]
An easy way to do this is to change the cards `--width` custom property. It ships with `fit-content` but you can target your cards and set it to something that suites your needs. 
essentially, you need to write css that targets the cards you want to be full width. The core components expose a variety of custom properties to help consumers override / customize what gets shipped by default. You can see the list for cards [here](https://clarity.design/storybook/core/?path=/story/components-card--page), in the API table at the bottom of the page. 

```css
    <style>
      cds-card {
        --width: 100%;
      }
    </style>
```

![Screen Shot 2021-07-15 at 2 48 14 PM](https://user-images.githubusercontent.com/433692/125862265-09429708-a676-4370-a324-f4ca10467e84.png)


[Here](https://stackblitz.com/edit/cds-card-100-width?file=index.html) is a simple example.

**Reply by mathisloge** [reply]
thanks!
